revolv/templates/project/project.html

```diff
@@ -1,240 +1,319 @@
 {% extends "base/base.html" %}
-
 {% load staticfiles %}
 {% load humanize %}
-
 {% block head %}
-    <script type="text/javascript">
-        window.PARTIAL_COMPLETENESS = {{ project.partial_completeness_as_js }};
-    </script>
+<script type="text/javascript">
+  window.PARTIAL_COMPLETENESS = {
```

Comment:
There's no need to format these `{` characters, they're part of the [Django template language](https://docs.djangoproject.com/en/1.9/topics/templates/). Please revert these changes.